commands/generate.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Alex Ellis 2017. All rights reserved.
```

Comment:
Given this is a new file, let's add the Copyright of `OpenFaaS Author(s)` which through the wording will acknowledge your copyright in this file.